### PR TITLE
Water-Coolers, Floor Lamps, and Benches

### PR DESCRIPTION
**Pull request by SinTwo**
Adds benches, floor lamps, and upgrades the Water-Coolers so they now can have a cup dispenser that holds up to 10 paper cups.

bench1

lamp

watercooler

### CODE REVIEW

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -203,13 +203,24 @@
...
 		..()
 		reagents.add_reagent("water",120)
-	else
-		icon_state = "water_cooler_0"
+	update_icon()
+
+/obj/structure/reagent_dispensers/water_cooler/examine(mob/user)
+	..()
+	if(cupholder)
+		user << "\blue There are [cups] cups in the cup dispenser."
```
Comment (Yoshax) [inline code/modules/reagents/reagent_dispenser.dm]: You should be using span classes.

**Comment by EmperorJon** [comment]
Can we also get a changelog considering it adds NEW THINGS.

**Comment by Yoshax** [comment]
Since they are children of tables, what happens when you build one next to a table?

**Comment by Hubblenaut** [comment]
I thought of adding benches once, but any character positioned on a turf with a vertically laid out bench will just look like they are STANDING on it, opposed to sitting...

**Comment by SinTwo** [comment]
@Yoshax Nothing! They build normally and do not try to connect.
@Hubblenaut Indeed it does; however, the same is also true for stools, no? (albeit to a lesser degree, but still!)
@EmperorJon Dammit, I knew I was forgetting something.

**Pull request merged**